services/api/service.go

```diff
@@ -170,6 +191,15 @@ type RelayAPI struct {
 
 	payloadAttributes     map[string]payloadAttributesHelper // key:parentBlockHash
 	payloadAttributesLock sync.RWMutex
+
+	// The slot we are currently optimistically simulating.
+	optimisticSlot uberatomic.Uint64
+	// The number of optimistic blocks being processed (only used for logging).
+	optimisticBlocksInFlight uberatomic.Uint64
+	// Wait group used to monitor status of per-slot optimistic processing.
+	optimisticBlocks sync.WaitGroup
```

Comment:
use `WG` suffix to make it clear it's a waitgroup

```suggestion
	optimisticBlocksWG sync.WaitGroup
```

Reply:
done!